xl-v1-multipart-common.go

```diff
@@ -256,6 +256,49 @@ func cleanupUploadedParts(bucket, object, uploadID string, storageDisks ...Stora
 	return nil
 }
 
+// Returns if the prefix is a multipart upload.
+func (xl xlObjects) isMultipartUpload(bucket, prefix string) bool {
```

Comment:
We should pick a random disk, it is not necessary to verify all the disks. It is expected to be correct. Use `getRandomDisk`